src/app/project/ontology/ontology.component.ts

```diff
@@ -618,7 +618,11 @@ export class OntologyComponent implements OnInit {
                         this._dspApiConnection.v2.onto.deleteResourceProperty(resProp).subscribe(
                             (response: OntologyMetadata) => {
                                 this.loading = false;
-                                this.resetOntology(this.ontologyIri);
+                                // this.resetOntology(this.ontologyIri);
```

Comment:
I would remove it if not needed anymore.

Reply:
definitely, removed in [3b60ad2](https://github.com/dasch-swiss/dsp-app/pull/831/commits/3b60ad2341140b76145e266b0195037a1a255df7)